pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go

```diff
@@ -47,6 +48,8 @@ type Config struct {
 	ExportClusterSettings bool `yaml:"cluster_settings,omitempty"`
 	// Export stats for shards in the cluster (implies indices).
 	ExportShards bool `yaml:"shards,omitempty"`
+	// Include informational aliases metrics
+	IncludeAliases bool `yaml:"aliases,omitempty"`
```

Comment:
Should the docs be updated to include the new field?

Reply:
Yes they should.

Reply:
I think it goes beyond breaking compilation though, we should see if there's any new options that the exporters expose that we should be propagated to our configs.

Reply:
Good call. There were 2 additional collectors with their flags. And a copy/paste error.